src/TuDa.CIMS.Web/Components/ShoppingCartFooter.razor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Components;
+using MudBlazor;
+using TuDa.CIMS.Shared.Entities;
+
+namespace TuDa.CIMS.Web.Components;
+
+/// <summary>
+/// Class for the shopping cart footer
+/// </summary>
+/// <param name="dialogService"></param>
+public partial class ShoppingCartFooter(IDialogService dialogService) : ComponentBase
```

Comment:
Is the dialogService intended for future use? Because github now says it isn't used.

Reply:
Oh yes I forgot to remove it. I tried to create an extra dialog for confirmation.